README.md

```diff
@@ -115,30 +115,28 @@ There are two forms of tests for the broker, the unit tests and the integration
 
 ### Running the unit tests
 
-To run the tests of this broker, you need a postgres and MySQL running locally, without SSL.
+To run the tests of this broker, you need a Postgres and MySQL running locally, without SSL.
 
 In travis we use the [postgres service](https://docs.travis-ci.com/user/database-setup/#PostgreSQL) and the [mysql service](https://docs.travis-ci.com/user/database-setup/#MySQL)
 
-The tests can read from environment variables the server configuration, with the following defaults:
- * `POSTGRESQL_HOSTNAME=localhost`
- * `POSTGRESQL_PORT=5432`
- * `POSTGRESQL_USERNAME=postgres`
- * `POSTGRESQL_PASSWORD=`
+The tests can read from environment variables to configure the connection
+details. See the `sqlengine` test suites for more details.
 
-You can run locally a postgres docker container and a homebrew-installed MySQL to run the tests
+You can use containers in [Docker Machine](https://docs.docker.com/machine/overview/):
 
 ```
+# Replace <VM_NAME> with the name from `docker-machine ls`
+DOCKER_IP=$(docker-machine ip <VM_NAME>)
+
 docker run -p 5432:5432 --name postgres -e POSTGRES_PASSWORD= -d postgres
-export POSTGRESQL_HOSTNAME=${DOCKER_HOSTNAME:-localhost} # Change accordingly
+export POSTGRESQL_HOSTNAME=${DOCKER_IP}
 
-brew install mysql
-mysql.server start
+docker run -p 3306:3306 --name mysql -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -d mysql
+export MYSQL_HOSTNAME=${DOCKER_IP}
```

Comment:
I think this will vary depending on whether you're using docker-machine or Docker for Mac. The latter works a lot more like Docker on linux, and hides the fact that there's an underlying VM much more.